src/SprykerEco/Zed/PunchoutCatalogs/Business/PunchoutCatalogsFacadeInterface.php

```diff
@@ -78,4 +80,36 @@ public function updateConnection(PunchoutCatalogConnectionTransfer $punchoutCata
      * @return \Generated\Shared\Transfer\PunchoutCatalogTransactionTransfer|null
      */
     public function findTransactionById(int $idPunchoutCatalogTransaction): ?PunchoutCatalogTransactionTransfer;
+
+    /**
+     * Specification:
+     * - Finds punchout catalogs which use given CompanyBusinessUnitTransfer.
+     * - Returns CompanyBusinessUnitResponseTransfer with check results.
+     * - CompanyBusinessUnitResponseTransfer::isSuccessful is equal to true when usage cases were not found, false otherwise.
+     *
+     * @api
+     *
+     * @param \Generated\Shared\Transfer\PunchoutCatalogConnectionTransfer $punchoutCatalogConnectionTransfer
+     *
+     * @return \Generated\Shared\Transfer\CompanyBusinessUnitResponseTransfer
+     */
+    public function isCompanyBusinessUnitDeletable(
```

Comment:
incoming parameter is not relevant to method name.
Please use CompanyBusinessUnitTransfer

Reply:
Done.